src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+const MALE_GENDER = 'm';
+const FEMALE_GENDER = 'f';
+const CENTURY_DURATION = 100;
+
+function countAverageAge(value) {
+  return value.reduce((sum, age) => sum + age, 0) / value.length;
+};
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const maleAges
+    = century === undefined
+      ? people
+        .filter(({ sex }) => sex === MALE_GENDER)
+        .map(({ died, born }) => died - born)
+      : people
+        .filter(({ sex, died }) => sex === MALE_GENDER
+        && Math.ceil(died / CENTURY_DURATION) === century)
+        .map(({ died, born }) => died - born);
```

Comment:
You don't need to compare with undefined, just `!century` is enough.
Also, you repeat `map` in both cases, DRY. You can do the `!century` check inside a callback so it reduces your code.